dotnet-estimation/dotnet-estimation/Templates/WebAPI/Devon4Net.Application.WebAPI.Implementation/Business/SessionManagement/Controllers/StatusController.cs

```diff
@@ -32,7 +32,7 @@ public StatusController(ISessionService SessionService)
         [ProducesResponseType(StatusCodes.Status400BadRequest)]
         [ProducesResponseType(StatusCodes.Status404NotFound)]
         [ProducesResponseType(StatusCodes.Status500InternalServerError)]
-        [Route("/estimation/v1/session/{id:long}/status")]
+        [Route("/estimation/v1/status/{id:long}")]
```

Comment:
I think you accidentally reverted @BenjaminE98's latest commit where he updated the route name.

Reply:
changed the code as suggested